### PR TITLE
Cross-mio updates for MAB

### PR DESCRIPTION
**Pull request by rtravis-whoi**
**Vocab.csv:**

- added PHSEN-G refdes to MAB surface moorings (refer to 1100-00006 attached)
- added 3 'new' MAB gliders - 335, 388, 514

**Eng bulk load csv:**

- 2 new additions for CNSM-2 (refer to CP10CNSM-00002 config sheet in Vault)

**Sensor bulk load csv:**

- New prawler sensors (for 16035-05) - Vault in Platform Records/Prawlers
- New WFP sensors (for 13103-01) - Vault in Platform Records/Coastal_WFPs (also photo to confirm SBE43 sensor SN 3412 because McLane didn't update the associated ballast sheet - which happens often). 

**Cruise info csv:**

- Added Pioneer 21 cruise AR85-02; estimated start/end dates of 2024-12-09 and 2024-12-22.

Also made a few tweaks to description fields as needed (I think it was mainly the prawler sensors in sensor bulk load).  

[1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-11-08_Ver_2-06.xlsx](https://github.com/user-attachments/files/17777308/1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-11-08_Ver_2-06.xlsx)
IMG_6542

[eng_bulk_load-AssetRecord_2024-11-15.xlsx](https://github.com/user-attachments/files/17777461/eng_bulk_load-AssetRecord_2024-11-15.xlsx)
[sensor_bulk_load-AssetRecord_2024-11-15.xlsx](https://github.com/user-attachments/files/17777462/sensor_bulk_load-AssetRecord_2024-11-15.xlsx)
[vocab_2024-11-15.xlsx](https://github.com/user-attachments/files/17777463/vocab_2024-11-15.xlsx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Pioneer 21 AM Updates](https://trello.com/c/65WTFp4J/252-pioneer-21-am-updates)

**Comment by rtravis-whoi** [comment]
[Pioneer 21 - CNSM SeapHOx](https://trello.com/c/aQhBDuVG/255-pioneer-21-cnsm-seaphox)

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -452,6 +453,7 @@ CGCON-NDCL26-00017,OL000328,Sensor,0,Near Surface Instrument Frame Data Concentr
 CGCON-NDCL26-00018,OL000265,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,0018,,,,,Woods Hole Oceanographic Institution
 CGCON-NDCL26-00020,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,20,,,,,
 CGCON-NDCL26-00021,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,21,,,,,
+CGCON-NDCL26-00022,,Sensor,0,Near Surface Instrument Frame Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,22,,,,,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/eng_bulk_load-AssetRecord.csv:456]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -1102,8 +1102,9 @@ CGINS-CTDPFL-00150,OL000039,Sensor,0,CTD Profiler: CTDPF Series L,CTDPF Series L
 CGINS-CTDPFL-00151,OL000043,Sensor,0,CTD Profiler: CTDPF Series L,CTDPF Series L,Sea-Bird Electronics,SBE 52-MP,151,,,,,
 CGINS-CTDPFL-00152,OL000051,Sensor,0,CTD Profiler: CTDPF Series L,CTDPF Series L,Sea-Bird Electronics,SBE 52-MP,152,,,,,
 CGINS-CTDPFL-00153,OL000055,Sensor,0,CTD Profiler: CTDPF Series L,CTDPF Series L,Sea-Bird Electronics,SBE 52-MP,153,,,,,
-CGINS-CTDPFP-00068,,Sensor,0,CTD Profiler: CTDPF Series P,,Sea-Bird Electronics,SBE 37-SM,709-0068,,,,,Woods Hole Oceanographic Institution
-CGINS-CTDPFP-00070,,Sensor,0,CTD Profiler: CTDPF Series P,,Sea-Bird Electronics,SBE 37-SM,709-0070,,,,,Woods Hole Oceanographic Institution
+CGINS-CTDPFP-00067,,Sensor,0,CTD Profiler: CTDPF Series P,CTDPF for Prawler,Sea-Bird Electronics,SBE 37-SM,709-0067,,,,,Woods Hole Oceanographic Institution
+CGINS-CTDPFP-00068,,Sensor,0,CTD Profiler: CTDPF Series P,CTDPF for Prawler,Sea-Bird Electronics,SBE 37-SM,709-0068,,,,,Woods Hole Oceanographic Institution
+CGINS-CTDPFP-00070,,Sensor,0,CTD Profiler: CTDPF Series P,CTDPF for Prawler,Sea-Bird Electronics,SBE 37-SM,709-0070,,,,,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:1107]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -1118,13 +1119,15 @@ CGINS-DOFSTK-02725,N00093,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series
...
 CGINS-DOFSTK-04361,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series K,DOFST for WFP Coastal ML14378-01,Sea-Bird Electronics,SBE 43F,43-4361,,20230201,,,Oregon State University
 CGINS-DOFSTK-04511,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series K,DOFST for WFP Coastal ML12991-03,Sea-Bird Electronics,SBE 43F,43-4511,,2024,,,Oregon State University
-CGINS-DOFSTP-04320,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series P,DOFST for Prawler,Aanderaa,Optode 4831,4320,,,,,Woods Hole Oceanographic Institution
-CGINS-DOFSTP-04321,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series P,DOFST for Prawler,Aanderaa,Optode 4831,4321,,,,,Woods Hole Oceanographic Institution
+CGINS-DOFSTP-04319,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series P,DOFST for Prawler,Aanderaa,Optode 4330F,4319,,,,,Woods Hole Oceanographic Institution
+CGINS-DOFSTP-04320,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series P,DOFST for Prawler,Aanderaa,Optode 4330F,4320,,,,,Woods Hole Oceanographic Institution
+CGINS-DOFSTP-04321,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series P,DOFST for Prawler,Aanderaa,Optode 4330F,4321,,,,,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:1130]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 4]
```diff
@@ -1620,6 +1623,7 @@ CGINS-FLORTO-01347,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series O
 CGINS-FLORTO-01348,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series O,,WET Labs,ECO Puck BB3,1348,,,,,Woods Hole Oceanographic Institution
 CGINS-FLORTO-01349,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series O,,WET Labs,ECO Puck BB3,1349,,,,,
 CGINS-FLORTO-07929,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series O,,WET Labs,ECO Puck BB3,7929,,,,,Woods Hole Oceanographic Institution
+CGINS-FLORTP-08570,,Sensor,1,3-Wavelength Fluorometer for Prawler: FLORT Series P,FLORT for Prawler,WET Labs,ECO BBFL2-w,8570,,,,,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:1626]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 5]
```diff
@@ -2240,6 +2244,7 @@ CGINS-PARADK-20609,,Sensor,0,Photosynthetically Available Radiation: PARAD Serie
 CGINS-PARADK-20610,,Sensor,0,Photosynthetically Available Radiation: PARAD Series K,,Biospherical Instruments,QSP-2200,20610,,,,,Woods Hole Oceanographic Institution
 CGINS-PARADK-20611,,Sensor,0,Photosynthetically Available Radiation: PARAD Series K,,Biospherical Instruments,QSP-2200,20611,,,,,Woods Hole Oceanographic Institution
 CGINS-PARADK-20612,,Sensor,0,Photosynthetically Available Radiation: PARAD Series K,,Biospherical Instruments,QSP-2200,20612,,,,,Woods Hole Oceanographic Institution
+CGINS-PARADK-20658,,Sensor,0,Photosynthetically Available Radiation: PARAD Series K,PAR for Coastal WFP,Biospherical Instruments,QSP-2200,20658,,,,,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:2247]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 6]
```diff
@@ -1118,13 +1119,15 @@ CGINS-DOFSTK-02725,N00093,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series
 CGINS-DOFSTK-02726,N00100,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series K,DOFST for WFP Coastal ML13103-04,Sea-Bird Electronics,SBE 43F,43-2726,3.35,20140128,,,Woods Hole Oceanographic Institution
 CGINS-DOFSTK-02978,N00030,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series K,DOFST for WFP Coastal ML12991-01,Sea-Bird Electronics,SBE 43F,43-2978,,20150713,92259.49,,Woods Hole Oceanographic Institution
 CGINS-DOFSTK-03370,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series K,DOFST for WFP Coastal ML12991-05,Sea-Bird Electronics,SBE 43F,43-3370,,,,,Woods Hole Oceanographic Institution
+CGINS-DOFSTK-03412,,Sensor,0,Dissolved Oxygen Fast Response: DOFST Series K,DOFST for Coastal WFP,Sea-Bird Electronics,SBE 43F,43-3412,,,,,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:1122]: looks good

**Review by cooleyky — APPROVED** [review]
looks good

File: bulk/eng_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (cdobs) [inline bulk/eng_bulk_load-AssetRecord.csv:456]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 2 above)
Comment (cdobs) [inline bulk/sensor_bulk_load-AssetRecord.csv:1107]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 3 above)
Comment (cdobs) [inline bulk/sensor_bulk_load-AssetRecord.csv:1130]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 6 above)
Comment (cdobs) [inline bulk/sensor_bulk_load-AssetRecord.csv:1122]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 5 above)
Comment (cdobs) [inline bulk/sensor_bulk_load-AssetRecord.csv:2247]: looks good

File: cruise/CruiseInformation.csv [hunk 7]
```diff
@@ -364,4 +364,5 @@ EK20240830,Elakha,2024-08-30T15:52:00,2024-08-30T19:47:00,Elakha CSPP deployment
 KH20240917,Kahuna,2024-09-17T14:00:00,2024-09-18T18:00:00,Pioneer glider cp_564 recovery
 SKQ202415S,Sikuliaq,2024-10-04T20:40:00,2024-10-18T21:30:00,Endurance 21
 KH20241008,Kahuna,2024-10-08T09:00:00,2024-10-08T20:00:00,Pioneer glider cp_339 recovery
-VA20241012,Virginia,2024-10-12T06:00:00,2024-10-14T06:00:00,Pioneer 20d AUV deployment
\ No newline at end of file
+VA20241012,Virginia,2024-10-12T06:00:00,2024-10-14T06:00:00,Pioneer 20d AUV deployment
+AR85-02,Neil Armstrong,2024-12-09T00:00:00,2024-12-22T23:59:00,Pioneer 21 (MAB)
```
Comment (cdobs) [inline cruise/CruiseInformation.csv:368]: looks good

File: vocab/vocab.csv [hunk 8]
```diff
@@ -1070,6 +1075,7 @@ CP11SOSM-RID26-01-PRTSZA000,Coastal Pioneer MAB,Southern Surface Mooring,Near Su
 CP11SOSM-RID26-03-TURBDA000,Coastal Pioneer MAB,Southern Surface Mooring,Near Surface Instrument Frame,Turbidity,Sea-Bird Electronics,ECO,7,7
 CP11SOSM-RID26-04-VELPTA000,Coastal Pioneer MAB,Southern Surface Mooring,Near Surface Instrument Frame,Single Point Velocity Meter,Nortek,Aquadopp 300m,7,7
 CP11SOSM-RID26-06-PHSEND000,Coastal Pioneer MAB,Southern Surface Mooring,Near Surface Instrument Frame,Seawater pH,Sunburst,SAMI-pH,7,7
+CP11SOSM-RID26-06-PHSENG000,Coastal Pioneer MAB,Southern Surface Mooring,Near Surface Instrument Frame,Seawater pH,Sea-Bird Electronics,Deep SeapHOx (350 m),7,7
```
Comment (cdobs) [inline vocab/vocab.csv:1078]: all phsen changes look good to me

File: vocab/vocab.csv [hunk 9]
```diff
@@ -1219,6 +1225,13 @@ CP15MOAS-G0913-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 913
...
 CP15MOAS-G0913-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 913,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP15MOAS-GL335,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 335,,,,0,200
+CP15MOAS-GL335-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 335,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,200
+CP15MOAS-GL335-01-ADCPAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 335,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,200
+CP15MOAS-GL335-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 335,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,200
+CP15MOAS-GL335-03-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 335,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,200
+CP15MOAS-GL335-04-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 335,Dissolved Oxygen,Aanderaa,Optode 4831,0,200
+CP15MOAS-GL335-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 335,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,200
```
Comment (cdobs) [inline vocab/vocab.csv:1234]: looks good

File: vocab/vocab.csv [hunk 10]
```diff
@@ -1247,13 +1260,28 @@ CP15MOAS-GL387-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387
...
 CP15MOAS-GL387-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP15MOAS-GL388,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 388,,,,0,1000
+CP15MOAS-GL388-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 388,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+CP15MOAS-GL388-01-ADCPAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 388,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CP15MOAS-GL388-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 388,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
+CP15MOAS-GL388-03-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 388,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP15MOAS-GL388-04-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 388,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP15MOAS-GL388-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 388,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
```
Comment (cdobs) [inline vocab/vocab.csv:1269]: looks good

File: vocab/vocab.csv [hunk 11]
```diff
@@ -1247,13 +1260,28 @@ CP15MOAS-GL387-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387
...
+CP15MOAS-PG514,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 514,,,,0,1000
+CP15MOAS-PG514-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 514,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+CP15MOAS-PG514-01-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 514,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP15MOAS-PG514-02-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 514,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP15MOAS-PG514-03-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 514,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
+CP15MOAS-PG514-04-FLORTO000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 514,3-Wavelength Fluorometer,WET Labs,ECO BB3,0,1000
+CP15MOAS-PG514-05-NUTNRM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 514,Nitrate,Satlantic,SUNA V2,0,1000
+CP15MOAS-PG514-06-PARADM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 514,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
```
Comment (cdobs) [inline vocab/vocab.csv:1284]: looks good

**Review by cdobs — APPROVED** [review]
looks good

**Pull request merged**